raiden_libs/transport/matrix.py

```diff
@@ -71,7 +74,7 @@ def dispatch(self, room, event):
                 self.run_message_callbacks(event['content']['body'])
                 log.debug("{0}: {1}".format(event['sender'], event['content']['body']))
 
-    def transmit_data(self, message):
+    def transmit_data(self, message: str, target_node: Address = None):
```

Comment:
will the target node always be an address? 
how about having a simple str instead, and validating it in a custom function of the `Transport`

```
def transmit_data(self, message: str, target_node: str):
    assert self.validate_target(str)  # or raise an exception or whatever
    ....

@staticmethod
def validate_target(target: str) -> Bool:
    return is_address(str)
```